docs/modules/lint-staged.md

```diff
@@ -0,0 +1,11 @@
+# lint-staged
+
+[lint-staged](https://github.com/lint-staged/lint-staged) can be replaced with smaller and faster alternatives
+
+# Replacements
+
+## nano-staged
+
+[nano-staged](https://github.com/usmanyunusov/nano-staged) offers the same API while being 142x smaller and slighty faster
```

Comment:
its probably safer to say `while being much smaller and slightly faster` since the size may change over time